payments server not starting on circle

See https://circleci.com/gh/mozilla/fxa/60907#tests/containers/2

From https://60907-36831081-gh.circle-artifacts.com/0/logs/payments-server-PORT-3031-error.log:

```
The following changes are being made to your tsconfig.json file:
  - compilerOptions.jsx must be preserve (JSX is compiled by Babel)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fxa-payments-server@1.143.1 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fxa-payments-server@1.143.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-08-12T16_54_31_467Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! fxa-payments-server@1.143.1 build-start-dev: `npm run build && npm run start-dev`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fxa-payments-server@1.143.1 build-start-dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-08-12T16_54_31_477Z-debug.log

```

--- a/packages/fxa-payments-server/src/store/utils.tsx
+++ b/packages/fxa-payments-server/src/store/utils.tsx
@@ -1,5 +1,5 @@
 import { ActionType as PromiseActionType } from 'redux-promise-middleware';
-import { Action, State, FetchState } from './types';
+import { Action, FetchState } from './types';
 
 type MappedObject = { [propName: string]: any };
 export const mapToObject = (

